.github/workflows/project_automation_set_in_progress.yml

```diff
@@ -0,0 +1,205 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Set PR and Linked Issues to In Progress
+# Possible TODO: expand queries and add mutations to set to current release and sprint  in addition to "In Progress"
+
+on:
+  pull_request_target:
+    # Run this action when a PR is opened or edited
+    # Issues do not have a graphQL connection to linked PRs so we can't use that event
+    types: [opened, edited]
+      
+env:
+ GH_TOKEN: ${{ secrets.PROJECT_MANAGEMENT_SECRET }}  # !!TODO: REPLACE WITH A TOKEN THAT HAS PROJECT MANAGEMENT PERMISSIONS!!
+ ORG: ${{ github.event.repository.owner.login }}
+ PR_NUMBER: ${{ github.event.pull_request.number }}
+ REPO: ${{ github.event.repository.name }}
+
+ # The environment vars below are hard-coded from external queries to save time + complexity here
+ # Note: PVT means Project V2, not "Private"
+ PROJECT_ID: "PVT_kwDOABpemM4AEhOI"
+ STATUS_FIELD_ID: "PVTSSF_lADOABpemM4AEhOIzgCmnYc"
+ IN_PROGRESS_PROJECT_OPTION_ID: "47fc9ee4"
+
+jobs:
+  query_and_mutate_project_fields:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Wait 1 Second
+        id: sleep
+        run: sleep 1
+        
+      - name: Get PR Project ID
+        id: get_pr_id
+        run: |
+            # Query up to 10 projects for the PR
+            gh api graphql -f query='
+              query {
+                organization(login: "${{ env.ORG }}") {
+                  repository(name: "${{ env.REPO }}") {
+                    issueOrPullRequest(number: ${{ env.PR_NUMBER }}) {
+                      ... on PullRequest {
+                        id
+                        projectItems(first: 10) {
+                          edges {
+                            node {
+                              id
+                              project {
+                                id
+                              }
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }' > project_data.json
+            
+            # Filter the json result to only the project-specific ID for the PR
+            # A PR can be in multiple projects so we need to filter by the project ID we want
+            pr_id=$(jq -r '.data.organization.repository.issueOrPullRequest.projectItems.edges[] |
+                           select(.node.project.id == "${{ env.PROJECT_ID }}") |
+                           .node.id' project_data.json)
```

Comment:
I'm surprised by how complicated the graphql API makes this operation. We need to query all the projects a PR could belong to and then filter to the specific `PROJECT_ID`? 

I would think it'd be simpler to get the id of the PR for a given project...